stream-video-android-ui-compose/src/main/kotlin/io/getstream/video/android/compose/ui/components/participants/ParticipantAvatars.kt

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package io.getstream.video.android.compose.ui.components.participants.internal
```

Comment:
I moved the public `ParticipantAvatars` component out of the `participants/internal` package. Extra handling needed?